### PR TITLE
chore:  docs kyverno comparison

### LINKED ISSUE #1009: Pepr vs Kyverno in the FAQ
### Is your feature request related to a problem? Please describe.
A community user suggested we add a comparison vs Kyverno in our FAQ. It it is a great idea to introduce new potential users to the product

### Describe the solution you'd like

- **Given** someone wants to learn about Pepr that is familiar with Kyverno 
- **When** they ask a question around the comparison
- **Then** we can point them to the docs

### Describe alternatives you've considered
(optional) A clear and concise description of any alternative solutions or features you've considered.

### PR DESCRIPTION
**Pull request by cmwylie19**
## Description

A suggestion in our Kubernetes Slack was to write a Kyverno comparison in the FAQ section of the docs 

## Related Issue

Fixes #1009

Relates to #

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Other (security config, docs update, etc)

## Checklist before merging

- [x] Test, docs, adr added or updated as needed
- [x] [Contributor Guide Steps](https://docs.pepr.dev/main/contribute/#submitting-a-pull-request) followed

### CODE REVIEW

File: docs/080_faq/README.md [hunk 1]
```diff
@@ -13,6 +13,26 @@ or
...
+
+Differences:
+* Pepr is more like a "framework" than a tool, in Pepr you create a Pepr [Module](https://docs.pepr.dev/main/user*guide/pepr*modules/), in the module itself you define [capabilities](https://docs.pepr.dev/main/user*guide/capabilities/) that enforce / apply desired cluster state
+* Pepr is TypeScript and Kyverno is Go
+* Pepr gives the flexibility of a full*fledged strongly typed programming language decide what decisions to make based on events happening in the cluster. IE, when this pod is created then i could theoretically create the same pod in another cluster. (You can do whatever you want because it is TypeScript)
+* There is a side of Pepr that is much more similar to Kube*Builder or Operator SDK, it can be used to reconcile events in order. You can apply a CustomResourceDefinition and control cluster state based on that custom resource.
+* * When I see a `WebApplication` resource is created I want to deploy a certain deployment, service, serviceAccount, networkPolicy, serviceMonitor, PrometheusRule, etc
+* * When I see a certain `WebApplication` resource is deleted I wanted to delete said resources
+* * When I see it is updated ..
```
Comment (cmwylie19) [inline docs/080_faq/README.md]:
```suggestion
* * When I see it is updated, then update the deployed Kubernetes resources
```

File: docs/080_faq/README.md [hunk 2]
```diff
@@ -13,6 +13,26 @@ or
 npx --node-options="--disable-warning=DEP0040" pepr [command]
 ```
 
+## How does Pepr compare to Kyverno?
+
+Pepr has similarities but ultimately is very different than Kyverno.
+
```
Comment (cmwylie19) [inline docs/080_faq/README.md:19]: mission statements are not the same.

Pepr is about making operators as easy as possible.

Kyverno is about reporting, not building operators

Comment (schaeferka) [reply docs/080_faq/README.md:19]: Added as first bullet point in differences section.

File: docs/080_faq/README.md [hunk 3]
```diff
@@ -7,18 +7,41 @@
...
 
+## How does Pepr compare to Kyverno?
+
+Although Pepr and Kyverno have similarities, Pepr is very different than Kyverno.
+
+Similarities:
+
+* Both have Mutating Webhooks that can dynamically change resources before admission
+* Both have Validating Webhooks to configure what can/cannot go through admission
+* Both utilize Kubernetes Informers under the hood for pre-existing cluster resources (ie, resources that have already gone through admission)
```
Comment (cmwylie19) [inline docs/080_faq/README.md]:
```suggestion
* Both provide a way to react to changes to pre-existing cluster resources (ie, resources that have already gone through admission)
```

File: docs/080_faq/README.md [hunk 4]
```diff
@@ -7,18 +7,41 @@
...
+
+Similarities:
+
+* Both have Mutating Webhooks that can dynamically change resources before admission
+* Both have Validating Webhooks to configure what can/cannot go through admission
+* Both provide a way to react to changes to pre-existing cluster resources (ie, resources that have already gone through admission)
+
+Differences:
+
+* They have very different mission statements. Pepr focuses on making operators as easy as possible. Kyverno focuses on reporting, not building operators.
```
Comment (cmwylie19) [inline docs/080_faq/README.md]:
```suggestion
```

File: docs/080_faq/README.md [hunk 5]
```diff
@@ -7,18 +7,41 @@
...
-or 
+or
 
 ```bash
 npx --node-options="--disable-warning=DEP0040" pepr [command]
 ```
 
+## How does Pepr compare to Kyverno?
+
+Although Pepr and Kyverno have similarities, Pepr is very different than Kyverno.
```
Comment (cmwylie19) [inline docs/080_faq/README.md]:
```suggestion
Although Pepr and Kyverno have similarities, Pepr is very different than Kyverno. They have very different mission statements. Pepr focuses on making operators as easy as possible. Kyverno focuses on reporting, not building operators.
```

File: docs/080_faq/README.md [hunk 6]
```diff
@@ -7,18 +7,40 @@
...
+Differences:
+
+* Pepr is more like a "framework" than a tool. In Pepr you create a Pepr [Module](https://docs.pepr.dev/main/user*guide/pepr*modules/). In the Pepr module you define [capabilities](https://docs.pepr.dev/main/user*guide/capabilities/) that enforce / apply desired cluster state.
+* Pepr is written in TypeScript. Kyverno is written in Go.
+* Pepr provides the flexibility of a full-fledged, strongly typed programming language to decide what decisions to make based on events happening in the cluster. With Kyverno, you are limited to the constraints of YAML.
+* Pepr can be used to reconcile events in order, similar to Kube-Builder or Operator SDK.
+* Pepr can apply a CustomResourceDefinition and control cluster state based on that custom resource. For example:
+  * When a `WebApplication` resource is created, deploy a certain deployment, service, serviceAccount, networkPolicy, serviceMonitor, PrometheusRule, etc.
+  * When a `WebApplication` resource is deleted, delete all the resources that were created for that `WebApplication`
+  * When a `WebApplication` is updated, update the deployed Kubernetes resources for that `WebApplication`
```
Comment (cmwylie19) [inline docs/080_faq/README.md]:
```suggestion
* Pepr can apply a CustomResourceDefinition and control cluster state based on that custom resource.
```

File: docs/080_faq/README.md [hunk 7]
```diff
@@ -7,18 +7,37 @@
...
+* Pepr provides the flexibility of a full-fledged, strongly typed programming language to decide what decisions to make based on events happening in the cluster. With Kyverno, you are limited to the constraints of YAML.
+* Pepr can be used to reconcile events in order, similar to Kube-Builder or Operator SDK.
+* Pepr can apply a CustomResourceDefinition and control cluster state based on that custom resource.
+
+Both Pepr and Kyverno are great tools. Which one to use for your project depends on your use case.
 
 ## How do I add custom labels to Pepr's Kubernetes manifests?
 
-During the build process, custom labels can be added the Kubernetes manifests that Pepr generates based on the Pepr section of the `package.json`. Currently, adding custom labels to `namespace` is supported.  
+During the build process, custom labels can be added the Kubernetes manifests that Pepr generates based on the Pepr section of the `package.json`. Currently, adding custom labels to `namespace` is supported.
```
Comment (cmwylie19) [inline docs/080_faq/README.md]:
```suggestion
During the build process, custom labels can be added the `pepr-system` namespace based on the `pepr` section of the `package.json`. Checkout the [Customizing with package.json](../030_user-guide/[120_customization.md#packagejson-configurations-table].
```

File: docs/080_faq/README.md [hunk 8]
```diff
@@ -7,18 +7,37 @@
...
+* Pepr provides the flexibility of a full-fledged, strongly typed programming language to decide what decisions to make based on events happening in the cluster. With Kyverno, you are limited to the constraints of YAML.
+* Pepr can be used to reconcile events in order, similar to Kube-Builder or Operator SDK.
+* Pepr can apply a CustomResourceDefinition and control cluster state based on that custom resource.
+
+Both Pepr and Kyverno are great tools. Which one to use for your project depends on your use case.
 
 ## How do I add custom labels to Pepr's Kubernetes manifests?
 
-During the build process, custom labels can be added the Kubernetes manifests that Pepr generates based on the Pepr section of the `package.json`. Currently, adding custom labels to `namespace` is supported.  
+During the build process, custom labels can be added the `pepr-system` namespace based on the `pepr` section of the `package.json`. Checkout the [Customizing with package.json](../030_user-guide/[120_customization.md#packagejson-configurations-table].
```
Comment (cmwylie19) [inline docs/080_faq/README.md]:
```suggestion
During the build process, custom labels can be added the `pepr-system` namespace based on the `pepr` section of the `package.json`. Checkout the [Customizing with package.json](../030_user-guide/[120_customization.md#packagejson-configurations-table).
```

File: docs/080_faq/README.md [hunk 9]
```diff
@@ -7,18 +7,37 @@
...
+* Pepr provides the flexibility of a full-fledged, strongly typed programming language to decide what decisions to make based on events happening in the cluster. With Kyverno, you are limited to the constraints of YAML.
+* Pepr can be used to reconcile events in order, similar to Kube-Builder or Operator SDK.
+* Pepr can apply a CustomResourceDefinition and control cluster state based on that custom resource.
+
+Both Pepr and Kyverno are great tools. Which one to use for your project depends on your use case.
 
 ## How do I add custom labels to Pepr's Kubernetes manifests?
 
-During the build process, custom labels can be added the Kubernetes manifests that Pepr generates based on the Pepr section of the `package.json`. Currently, adding custom labels to `namespace` is supported.  
+During the build process, custom labels can be added the `pepr-system` namespace based on the `pepr` section of the `package.json`. Checkout the [Customizing with package.json](../030_user-guide/[120_customization.md#packagejson-configurations-table).
```
Comment (cmwylie19) [inline docs/080_faq/README.md]:
```suggestion
During the build process, custom labels can be added the `pepr-system` namespace based on the `pepr` section of the `package.json`. Checkout the [Customizing with package.json](../030_user-guide/120_customization.md#packagejson-configurations-table).
```

File: docs/080_faq/README.md [hunk 10]
```diff
@@ -7,18 +7,37 @@
...
+
+Similarities:
+
+* Both have Mutating Webhooks that can dynamically change resources before admission
+* Both have Validating Webhooks to configure what can/cannot go through admission
+* Both provide a way to react to changes to pre-existing cluster resources (ie, resources that have already gone through admission)
+
+Differences:
+
+* Pepr is more like a "framework" than a tool. In Pepr you create a Pepr [Module](https://docs.pepr.dev/main/user*guide/pepr*modules/). In the Pepr module you define [capabilities](https://docs.pepr.dev/main/user*guide/capabilities/) that enforce / apply desired cluster state.
```
Comment (cmwylie19) [inline docs/080_faq/README.md]:
```suggestion
* Pepr is more like a "framework" than a tool. In Pepr you create a Pepr [Module](../030_user-guide/020_pepr-modules.md). In the Pepr module you define [capabilities](../030_user-guide/040_capabilities.md) that enforce / apply desired cluster state.
```

File: docs/080_faq/README.md [hunk 11]
```diff
@@ -7,18 +7,37 @@
...
+* Pepr provides the flexibility of a full-fledged, strongly typed programming language to decide what decisions to make based on events happening in the cluster. With Kyverno, you are limited to the constraints of YAML.
+* Pepr can be used to reconcile events in order, similar to Kube-Builder or Operator SDK.
+* Pepr can apply a CustomResourceDefinition and control cluster state based on that custom resource.
+
+Both Pepr and Kyverno are great tools. Which one to use for your project depends on your use case.
 
 ## How do I add custom labels to Pepr's Kubernetes manifests?
 
-During the build process, custom labels can be added the Kubernetes manifests that Pepr generates based on the Pepr section of the `package.json`. Currently, adding custom labels to `namespace` is supported.  
+During the build process, custom labels can be added the `pepr-system` namespace based on the `pepr` section of the `package.json`. Checkout the [Customizing with package.json](../030_user-guide/120_customization.md#packagejson-configurations-table).
```
Comment (cmwylie19) [inline docs/080_faq/README.md]:
```suggestion
During the build process, custom labels can be added the `pepr-system` namespace based on the `package.json`. Checkout the [Customizing with package.json](../030_user-guide/120_customization.md#packagejson-configurations-table).
```

**Review by btlghrants — APPROVED** [review]

**Pull request merged**